MSGraphSamples.WPF/Helpers/TokenCacheHelper.cs

```diff
@@ -13,40 +13,42 @@ static class TokenCacheHelper
     {
         private static readonly object FileLock = new object();
 
-        private static readonly string LocalAppData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
-        private static readonly string ProjectName = Assembly.GetCallingAssembly().GetName().Name ?? "tokencache";
-        private static readonly string CacheDirectoryPath = $"{LocalAppData}\\{ProjectName}\\";
-        private static readonly string CacheFileName = "msalcache.bin";
-        private static string CacheFilePath => Path.Combine(CacheDirectoryPath, CacheFileName);
+        //private static readonly string LocalAppData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+        //private static readonly string ProjectName = Assembly.GetCallingAssembly().GetName().Name ?? "tokencache";
+        //private static readonly string CacheDirectoryPath = $"{LocalAppData}\\{ProjectName}\\";
+        //private static readonly string CacheFileName = "msalcache.bin";
+        //private static string CacheFilePath => Path.Combine(CacheDirectoryPath, CacheFileName);
 
-        public static void BeforeAccessNotification(TokenCacheNotificationArgs args)
-        {
-            if (!File.Exists(CacheFilePath))
-                return;
+        private static readonly string CacheFilePath = System.Reflection.Assembly.GetExecutingAssembly().Location + ".msalcache.bin";
```

Comment:
@kalyankrishna1 this is not compatible with MSIX Packaging, see: https://github.com/MicrosoftDocs/azure-docs/issues/49182
Please revert.